ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Ledger/SupportsMempool.hs

```diff
@@ -77,7 +87,7 @@ class ( UpdateLedger blk
           -> SlotNo -- ^ Slot number of the block containing the tx
           -> GenTx blk
           -> TickedLedgerState blk ValuesMK
-          -> Except (ApplyTxErr blk) (TickedLedgerState blk TrackingMK, Validated (GenTx blk))
+          -> Except (ApplyTxErr blk) (TickedLedgerState blk DiffMK, Validated (GenTx blk))
```

Comment:
Note to reviewer: `applyTx` now returns `DiffMK`. It is responsibility of the caller to do something with this. In particular, the IS doesn't care about the values, so `Diff` is good enough for normal operation of consensus